testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -2950,4 +2950,34 @@ QUnit.module('itemRendered event', () => { // T906117
             assert.equal(expectedItemsArray.length, 0);
         });
     });
+
+    QUnit.test('itemRendered callback is called for menu & treeview items, adaptivityEnabled: true (T1092214)', function(assert) {
+        assert.expect(11);
+        const onItemRenderedHandler = sinon.stub();
+
+        $('#menu').dxMenu({
+            dataSource: testDataSource,
+            adaptivityEnabled: true,
+            width: 50,
+            onItemRendered: onItemRenderedHandler
+        }).dxMenu('instance');
+
+        const checkRenderedItem = (call, itemText, itemClass) => {
+            assert.strictEqual(onItemRenderedHandler.getCall(call).args[0].itemData.text, itemText);
+            assert.ok($(onItemRenderedHandler.getCall(call).args[0].itemElement).hasClass(itemClass));
+        };
+
+        assert.strictEqual(onItemRenderedHandler.callCount, 2);
+        checkRenderedItem(0, 'item1', DX_MENU_ITEM_CLASS);
+        checkRenderedItem(1, 'item1', DX_TREEVIEW_ITEM_CLASS);
+
+        const $treeview = $('#menu').find(`.${DX_TREEVIEW_CLASS}`);
+        $treeview.find(`.${DX_TREEVIEW_ITEM_CLASS}`).eq(0).trigger('dxclick');
+        assert.strictEqual(onItemRenderedHandler.callCount, 3);
+        checkRenderedItem(2, 'item1_1', DX_TREEVIEW_ITEM_CLASS);
+
+        $treeview.find(`.${DX_TREEVIEW_ITEM_CLASS}`).eq(1).trigger('dxclick');
```

Comment:
we can move `$treeview.find(`.${DX_TREEVIEW_ITEM_CLASS}`)` to a variable to reduce copypaste

Reply:
We can get the second item after click on the first. So, we need additional 'find()' call to get actual items after click.